<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Academic and Professional Profile</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cerulean.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About Me</a>
</li>
<li>
  <a href="flexidashboard.html">NYC Restaurant Inspections Flexdashboard</a>
</li>
<li>
  <a href="mailto:&lt;singh.anmol@columbia.edu&gt;">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/asingh100/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Academic and Professional Profile</h1>

</div>


<p>Anmol Singh</p>
<p>9462 Winding Hill Way Lone Tree, CO 80124</p>
<p>Mobile: 3038086032 E-mail: <a href="mailto:singh.anmol@columbia.edu" class="email">singh.anmol@columbia.edu</a></p>
<p><strong>Personal Profile</strong></p>
<p>I am currently a master’s student in biostatistics at Columbia University conducting research in the field of statistical genetics with research experience in both academia and industry. My research and work experiences in conjunction with my coursework have made me an excellent problem solver, communicator, team player, and scientific writer. Furthermore, these experiences have given me valuable technical skills such as using machine learning algorithms, performing statistical analysis of data, and programming in a variety of languages including R, Python, Java, and MATLAB. I am currently interested in conducting research in one of the following areas: creating better screening and diagnostic tools for disease based on a person’s genome, identifying biomarkers that can cause the onset of disease, and creating personalized treatment methods based a variety of factors including their genetics. I look forward to applying the skills that I have learnt to a lab that is as passionate as I am about using the intersection of mathematics, computer science, and biology to help improve treatment methods, medical diagnoses, and patients’ lives.</p>
<p><strong>Honors and Awards</strong></p>
<p>➢ Distinguished Scholar Award: Fall 2016</p>
<p>o $108,000 scholarship to attend Tulane University</p>
<p>➢ Dean’s List: Spring 2019, Fall 2020, Spring 2020</p>
<p>o Awarded to students who have a 3.667 GPA or higher in a semester</p>
<p>➢ Summer Research Grant: Summer 2018</p>
<p>o $1,600 grant from the Busschaert lab to work on a collaborative research project</p>
<p><strong>Education</strong></p>
<p>September 2020 – May 2022 <strong>Master of Science, Biostatistics</strong></p>
<p>Columbia University, New York, NY</p>
<p>➢ Track: Statistical Genetics</p>
<p>August 2016 – May 2020 <strong>Bachelor of Science, Biomedical Engineering</strong></p>
<p>Tulane University, New Orleans, LA</p>
<p>➢ Thesis: Enhancing the Selectivity of Optical Anion Sensors Using Liposome Encapsulation and Anion Transporters</p>
<p>➢ Advisor: Dr. Nathalie Busschaert</p>
<p>➢ Major GPA: 3.556/4.0</p>
<p>➢ Minors: Mathematics and Business</p>
<p><strong>Research Experience</strong></p>
<p>August 2020 – Present <strong>Research Assistant in Neurogenomics</strong></p>
<p>Columbia University Irving Medical Center, New York, NY</p>
<p>➢ Advisors: Dr. Suzanne Leal and Dr. Gao Wang</p>
<p>➢ Using deep convolutional neural networks to identify genetic variants that could have an effect on the onset of Alzhimer’s disease and intelectual disability.</p>
<p>April 2020 – August 2020 <strong>Research Assistant in Pharmacogenomics</strong></p>
<p>Northwestern University Feinberg School of Medicine, Chicago, IL</p>
<p>➢ Advisor: Dr. Minoli Perera</p>
<p>➢ Created a novel method using the LAMatrix R package to incorporate DNA Methylation as a locus-specific covariate in eQTL mapping in African Americans</p>
<p>➢ Showed how incorporating methylation into eQTL mapping can potentially identify disease causing genetic variants such as variants related to the expression of GSTM3 and HSPA6, which are genes that have been linked to the occurrence of Hepatocellular Carcinoma.</p>
<p>➢ Published a first author journal article and gave a talk on this research in the 2021 Pacific Symposium on Biocomputing meeting.</p>
<p>January 2020 – August 2020 <strong>Research Assistant in Cancer Diagnostics</strong></p>
<p>Tulane University, New Orleans, LA</p>
<p>➢ Advisor: Dr. Quincy Brown</p>
<p>➢ Working on a project to use Convolutional Neural Networks to analyze prostate cancer histology images and determine which topological features correspond to different stages of prostate cancer</p>
<p>➢ Working on improving the code efficiency and organization by organizing each method into distinct packages which can then be imported into the main file for use</p>
<p>July 2018 – May 2020 <strong>Research Assistant in Medicinal Chemistry</strong></p>
<p>Tulane University, New Orleans, LA</p>
<p>➢ Advisor: Dr. Nathalie Busschaert</p>
<p>➢ Worked on an independent project to create a selective sensor for the acetate ion as a potential cancer therapeutic for my undergraduate thesis</p>
<p>➢ Worked on a collaborative project with a PhD student in the lab, Sarah Marshall, to create a selective sensor for the iodide ion using partially selective anion transporters and liposomes which has now been published in Chemical Communications</p>
<p>➢ Conducted spectroscopic assays with various anions and anion transporters to gauge the selectivity of each anion / transporter combination to find the best combination</p>
<p>➢ Performed the data manipulation and cleanup of the spectroscopic assay results using Origin and Microsoft Excel to prepare the experimental data for analysis</p>
<p>➢ Used linear and exponential regression models to estimate the rate of fluorescence change of the sensor in the presence of each anion to determine which anion transporter only causes a significant change for the anion we want to be detected</p>
<p>May 2015 – August 2015 <strong>Research Assistant in Medical Physics</strong></p>
<p>University of Chicago, Chicago, IL</p>
<p>➢ Advisor: Dr. Patrick La Riviere</p>
<p>➢ Used Python to assist with research on modeling to find optimal wavelengths for opto-acoustic imaging of breast tumors</p>
<p><strong>Work Experience</strong></p>
<p>January 2020 - April 2020 <strong>Data Analyst</strong></p>
<p>Tulane University Department of Sociology, New Orleans, LA</p>
<p>➢ Assisted with analyzing and compiling data for two different research projects, one on analyzing blogging trends from 2006 to 2017 and one on analyzing injustices that occur in the Chicago housing market</p>
<p>➢ Used R to combine data from multiple Excel files into one to facilitate more efficient data analysis</p>
<p>➢ Used R to create a linear regression model to predict how many blog posts would be posted in future years using a data set of the number of blog posts per year from 2008 to 2017</p>
<p>➢ Used R to analyze waiting list data from 2016, 2018, and 2020 to show how long it took for families to receive a grant from the government for emergency home repairs in Chicago</p>
<p>May 2018 - Present <strong>Academic Tutor</strong></p>
<p>Varsity Tutors, Online</p>
<p>➢ Tutors students struggling with mathematics to prepare them for the ACT, SAT, PSAT, and SAT II tests</p>
<p>➢ Facilitates learning by tailoring one-on-one sessions according to each student’s level</p>
<p>June 2019 – August 2019 <strong>R&amp;D Engineering Intern</strong></p>
<p>Allosource, Centennial, CO</p>
<p>➢ Used Python to create the Graphical User Interface for the automation of their skin processing machine</p>
<p>➢ Wrote a protocol that improves the existing decellularization technique for skin tissue thicker than 3.3 mm</p>
<p>with a 218% greater reduction in skin tissue cells when compared to the current decellularization method</p>
<p>➢ Used R to implement a linear regression model to estimate values for the DNA remaining on the skin tissue after decellularization was performed on the tissue</p>
<p>October 2017 – May 2018 <strong>Data Analytics Intern</strong></p>
<p>Forward Science, New Orleans, LA</p>
<p>➢ Assisted with marketing research using surveys for their products</p>
<p>➢ Distributed surveys across Tulane University’s campus to collect data</p>
<p>➢ Performed data analysis and created visualizations of the survey data using Microsoft Excel to generate conclusions about college students’ interest in their products</p>
<p><strong>Journal Articles</strong></p>
<p>➢ Singh, Anmol, Zhong, Yizhen, Park, Sehwan C., De, Tanima, Alcorn Cristina, and Perera, Minoli, Incorporation of DNA methylation into eQTL mapping in African Americans, Pacific Symposium on Biocomputing (2021), Accepted.</p>
<p>➢ Marshall, Sarah R, Singh, Anmol, Wagner, Jacqueline N, and Busschaert, Nathalie, “Enhancing the selectivity of optical sensors using synthetic transmembrane ion transporters”, Chemical Communications (2020), Accepted.</p>
<p><strong>Presentations</strong></p>
<p>Oral Presentations:</p>
<p>➢ Singh, Anmol, “Incorporation of DNA Methylation into eQTL Mapping In African Americans”, Pacific Symposium on Biocomputing. January 7, 2021. Kona,HI.</p>
<p>➢ Singh, Anmol, “Enhancing the Selectivity of Optical Anion Sensors Using Liposome Encapsulation and Anion Transporters”, Tulane University. Department of Biomedical Engineering Undergraduate Research Symposium. January 27, 2020. New Orleans, LA.</p>
<p>➢ Singh, Anmol, “Converting a non-selective halide sensor into a selective iodide sensor using liposomes and transporters”, Tulane University. Department of Chemistry Undergraduate Research Symposium. December 8, 2018. New Orleans, LA.</p>
<p>Poster Presentations:</p>
<p>➢ Marshall, Sarah Ruth, Singh, Anmol, Wagner, Jacqueline, and Busschaert, Nathalie, “Converting a nonselective halide sensor into a selective iodide sensor using liposomes and transporters”, 14th Annual International Symposium on Supramolecular Chemistry. June 2, 2019. Lecce, Italy.</p>
<p><strong>Skills</strong></p>
<p>➢ Programming Languages: R, SQL, MATLAB, Python, Java, C, C++</p>
<p>➢ Other Software: Tableau, Microsoft Office, SPSS, Solidworks, Origin Data Analysis Software</p>
<p>➢ Technical: Mathematical Modeling, Machine Learning, Data Analysis and Interpretation, Data Visualization, Statistical Analysis, Good Lab Practices, Scientific Writing</p>
<p>➢ Intrapersonal: Oral Communication, Written Communication, Teamwork</p>
<p>➢ Languages: English (Fluent), Hindi (Advanced), Punjabi (Advanced), Spanish (Beginner)</p>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
